capella/setup/disable_semantic_browser_auto_refresh.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+import fileinput
+import logging
+import os
+import pathlib
+import sys
+
+from lxml import etree
+
+NS_MAP = {
+    "xmi": "http://www.omg.org/XMI",
+    "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+    "basic": "http://www.eclipse.org/ui/2010/UIModel/application/ui/basic",
+    "advanced": (
+        "http://www.eclipse.org/ui/2010/UIModel/application/ui/advanced"
+    ),
+    "application": "http://www.eclipse.org/ui/2010/UIModel/application",
+    "menu": "http://www.eclipse.org/ui/2010/UIModel/application/ui/menu",
+}
+XPATH_EXPR = "//persistedState[@key='memento' and contains(@value, 'listeningToWorkbenchPageSelectionEvents')]"
+WORKSPACE_DIR = os.getenv("WORKSPACE_DIR", "/workspace")
+
+logging.basicConfig(level=os.getenv("LOG_LEVEL", "INFO"))
+logger = logging.getLogger(__file__)
+
+
+def check_environment_variable() -> None:
+    disable_semantic_browser = os.getenv(
+        "CAPELLA_DISABLE_SEMANTIC_BROWSER_AUTO_REFRESH", "0"
+    )
+    if disable_semantic_browser.lower() not in (
+        "1",
+        "true",
+        "t",
+    ):
+        logger.info(
+            "Environment variable `CAPELLA_DISABLE_SEMANTIC_BROWSER_AUTO_REFRESH` is set to `%s`. "
+            "The semantic browser auto refresh will not be disabled. ",
+            disable_semantic_browser,
+        )
+        sys.exit(0)
+
+    logger.debug(
```

Comment:
```suggestion
    logger.info(
```
IMHO: Same type of information with same relevance should have same logging level.

Example:
When the disabling is active, we want to know that.
Especially for the case where the action of replacement itself goes wrong and is not logging anything.